### PR TITLE
Added if_exists flag when drop index of edge and tag

### PR DESCRIPTION
**Pull request by wfystx**

### CODE REVIEW

File: src/graph/test/IndexTest.cpp [hunk 1]
```diff
@@ -129,6 +129,18 @@ TEST_F(IndexTest, TagIndex) {
         code = client->execute(query, resp);
         ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);
     }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string query = "DROP TAG INDEX not_exists_tag_index";
+        auto code = client->execute(query, resp);
+        ASSERT_NE(cpp2::ErrorCode::SUCCEEDED, code);
```
Comment (darionyaphet) [inline src/graph/test/IndexTest.cpp]: `ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);`

File: src/graph/test/IndexTest.cpp [hunk 2]
```diff
@@ -237,6 +249,18 @@ TEST_F(IndexTest, EdgeIndex) {
         code = client->execute(query, resp);
         ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);
     }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string query = "DROP EDGE INDEX not_exists_edge_index";
+        auto code = client->execute(query, resp);
+        ASSERT_NE(cpp2::ErrorCode::SUCCEEDED, code);
```
Comment (darionyaphet) [inline src/graph/test/IndexTest.cpp]: `ASSERT_EQ(cpp2::ErrorCode::E_EXECUTION_ERROR, code);`

**Comment by darionyaphet** [comment]
LGTM

**Comment by darionyaphet** [comment]
jenkins go

File: src/meta/processors/indexMan/DropTagIndexProcessor.cpp [hunk 3]
```diff
@@ -18,7 +18,11 @@ void DropTagIndexProcessor::process(const cpp2::DropTagIndexReq& req) {
     auto tagIndexID = getTagIndexID(spaceID, indexName);
     if (!tagIndexID.ok()) {
         LOG(ERROR) << "Tag Index not exist Space: " << spaceID << " Index name: " << indexName;
```
Comment (Amber1990Zhang) [inline src/meta/processors/indexMan/DropTagIndexProcessor.cpp]: not exists in Space

**Comment by jude-zhu** [comment]
relate #460

**Pull request merged**